extension.yaml

```diff
@@ -86,6 +89,21 @@ params:
     example: 'name,category,price,sales'
     required: false
 
+  - param: FORCE_DATA_SYNC
+    label: Force Data Sync
+    description: >-
+      The extension uses approaches to mitigate race conditions but it might not be a 100% fool proof.  This setting
+      will make sure the function is using the latest data from Firestore by making another read call before processing
+      and sending to Algolia.  NOTE: This will be a add to your usage of Firestore.
+    type: select
+    options:
+      - label: No
+        value: no
+      - label: Yes
+        value: yes
```

Comment:
as this is yaml, those values are actually the booleans `true` and `false`, meaning the code checking for the string `'yes'` will not work. You can either change these to strings or change the code to booleans

Reply:
the extension yaml doesn't support boolean types hence this approach.  I originally wanted to use booleans but did not have that option.

Reply:
also, I will get 'yes' and 'no'.  i tested this both locally and in Cloud Firebase and I am using strict equality.